### PR TITLE
Adding new FCU type

### PR DESCRIPTION
**Pull request by ccquigley**
for crac unit with return control. No fan points available

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -1452,6 +1452,20 @@ FCU_DFSS_DSP_HWZC_CHWZC_CO2M:
...
   - CHWZC
   - CO2M
+
+FCU_RTC_RHC_RHDHC_HTRC_DXRC_ZTM:
+  guid: "cc4f57f8-2fa4-4ac1-b32b-76ce5671dd0d"
+  description: "Fan coil with zone temp monitoring, return temp and humidity control, dx cooling, heating, and dehumidication and humidifcation modes."
+  is_canonical: true
+  implements:
+  - FCU
+  - RTC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1462]: This is redundant, you already have HTRC and DXRC

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -1452,6 +1452,20 @@ FCU_DFSS_DSP_HWZC_CHWZC_CO2M:
...
+  - RHC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1463]: this is redundant

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 3]
```diff
@@ -1452,6 +1452,18 @@ FCU_DFSS_DSP_HWZC_CHWZC_CO2M:
   - HWZC
   - CHWZC
   - CO2M
+
+FCU_RHDHC_HTRC_DXRC_ZTM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1456]: Where are the fan status points?

Comment (ccquigley) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1456]: This is the crac unit with no fan points available in the bms.

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**